### PR TITLE
Readme - how to deploy the operator from a custom image

### PR DESCRIPTION
**Pull request by jjaferson**
# Description

Adds how to deploy the operator from a custom image to the readme.

### CODE REVIEW

**Comment by Boomatang** [comment]
I don't know, I only found out about this command. Can you use the `IN_PROW` flag with this?

File: README.md [hunk 1]
```diff
@@ -114,6 +114,13 @@ Once the installation completed the command wil result in following output:
 "complete"
 ```
 
+### 6. Deploy a custom image of the operator
```
Comment (laurafitzgerald) [inline README.md:117]: I would say this isn't step 6. of the install but rather an alternative to step 4 of running the operator.

Comment (jjaferson) [reply README.md:117]: Yep, it make more sense actually.

**Comment by jjaferson** [comment]
> I don't know, I only found out about this command. Can you use the `IN_PROW` flag with this?

there is a section for that already https://github.com/integr8ly/integreatly-operator#3-configuration-optional

**Comment by Boomatang** [comment]
> > I don't know, I only found out about this command. Can you use the `IN_PROW` flag with this?
> 
> there is a section for that already https://github.com/integr8ly/integreatly-operator#3-configuration-optional

I can expect that answer

**Pull request merged**